### PR TITLE
skip test which depends on thinclient.jar for slim image

### PR DESCRIPTION
**Pull request by vanajamukkara**
This PR contains changes ..

- remove call to testAdminT3ChannelWithJms which depends on thinclient.jar, so that the integration tests can be run on slim images too. See JIRA OWLS-76081
- fix readme format
- setupenv.sh change - remove k8s installation step from tests for Jenkins run as its done in the Jenkins configuration before the tests are called
- setupenv.sh change - remove code which pulls the WL image to copy the thinclient jar as its not needed now.

### CODE REVIEW

**Comment by vanajamukkara** [comment]
external jenkins is clean - http://build.weblogick8s.org:8080/job/weblogic-kubernetes-operator-quicktest/474/console

File: integration-tests/README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ This documentation describes the functional use cases that are covered in integr
 The tests currently run in three modes: "shared cluster", "Jenkins", and "standalone" Oracle Linux, where the mode is controlled by the `SHARED_CLUSTER` and `JENKINS` environment variables described below. The default is "standalone".
 
 * "Standalone" Oracle Linux, i.e, run the tests manually with the `mvn` command.
-* Shared cluster - http://build.weblogick8s.org:8080/job/weblogic-kubernetes-operator-quicktest/
-* Jenkins - http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/ - Jenkins Run is restricted to Oracle Internal development process.
+* Shared cluster(remote k8s cluster) - http://build.weblogick8s.org:8080/job/weblogic-kubernetes-operator-quicktest/
+* Jenkins - http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/ - Jenkins Run is restricted to Oracle Internal development process. K8S is installed for every run.
```
[the remark this answers is not in the document]
Comment (vanajamukkara) [reply integration-tests/README.md]: ok, I removed the details.

**Comment by vanajamukkara** [comment]
> 
> 
> hmm - this is borderline - since the slim image and 12.2.1.4 are not released, I am not sure that we should be referencing them at all in here.. let me get advice on this before we merge it

Discussed with @anpanigr , will be commenting the test that depends on thinclient.jar as its not a generic use case that works for all images and create a JIRA to replace the use case with alternative solution. So, there will not be any reference to PS4 and slim image.

**Comment by markxnelson** [comment]
ok great.

File: integration-tests/src/test/resources/setupenv.sh [hunk 2]
```diff
@@ -164,6 +116,7 @@ function pull_tag_images_jrf {
   set -x
 }
 
+# the below function is not used
```
Comment (sankarpn) [inline integration-tests/src/test/resources/setupenv.sh]: If its not used we should remove it.

Comment (vanajamukkara) [reply integration-tests/src/test/resources/setupenv.sh]: done

**Comment by vanajamukkara** [comment]
internal jenkins is running - http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2089/console

**Pull request merged**